pyproject.toml

```
@@ -118,7 +118,8 @@ exclude_also = [
     ]
 
 [tool.setuptools.packages.find]
-exclude = ["tests", "results"]
+where = ["."]
+include = ["mteb*"]
```

```suggestion
where = ["."]
include = ["mteb"]
```
Do we need the star here, I could imagine that we could accidentally reintroduce this problem with a new folder. Otherwise, I think this is good.

---

I think we can also add `namespace=False` https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#finding-simple-packages

> When using tool.setuptools.packages.find in pyproject.toml, setuptools will consider [implicit namespaces](https://peps.python.org/pep-0420/) by default when scanning your project directory. To avoid pkg.namespace from being added to your package list you can set namespaces = false. This will prevent any folder without an __init__.py file from being scanned.

---

> I think we may also add `namespace=False` https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#finding-simple-packages

Honestly, I'm not familiar with setuptools, maybe we can deal with it when needed. 

So far, so good.

---

@KennethEnevoldsen I've tested and `.*` is needed to include all subdirs. Do you agree with this?

---

I tested it too, .* is needed.